### DISCUSSION TITLE
Why should I use cell.render("Cell") instead of cell.value ?

### DISCUSSION THREAD

**Discussion by leodevbro** [TanStack/table#3706] [General] [answered]
Hello, I'm confused, what's the point of `cell.render("Cell")` when I can just write `cell.value`?
I see in my browser that it renders the same way. So, why should I use the first one?

![2022-02-11_22-15-41------](https://user-images.githubusercontent.com/54504880/153647633-9fa0501a-3447-464e-9250-0eb56421acc1.png)

**Comment by tannerlinsley** [comment, marked as answer]
The render call formats your cell using the columns Cell formatter.

**Comment by leodevbro** [comment]
@tannerlinsley, Thanks. One more question: For some unusual reason, I want the cells to render their column name instead of cell value. Can I easily get the column name of the cell without this long workaround?

`<td>{tableColumns.find((x) => x.accessor === cell.column.id)?.Header}</td>`

That's because I found `cell.column.id` but I cannot find anything like this: `cell.column.name`.

**Comment by tannerlinsley** [comment]
cell.column.render(”Header”)

Tanner Linsley

**Reply by leodevbro** [reply]
Thanks so much, it works.
